app/models/decidim/vocdoni/election.rb

```diff
@@ -66,10 +66,59 @@ def minimum_answers?
       questions.any? && questions.all? { |question| question.answers.size > 1 }
     end
 
+    # Public: Checks if the election is ready for the questions step
+    #
+    # Returns a boolean.
+    def ready_for_questions_form?
+      present?
+    end
+
+    # Public: Returns the status of the election census
+    def census_status
+      @census_status ||= CsvCensus::Status.new(self)&.name
+    end
```

Comment:
I think this method can be private, as it's only used by other methods of this class. If that's correct, let's make it private explicitly, please 😄